rslib/src/scheduler/fsrs/weights.rs

```diff
@@ -161,16 +163,12 @@ fn single_card_revlog_to_items(
     entries.retain(|entry| {
         let manually_rescheduled =
             entry.review_kind == RevlogReviewKind::Manual || entry.button_chosen == 0;
-        let cram = entry.review_kind == RevlogReviewKind::Filtered;
+        let cram = entry.review_kind == RevlogReviewKind::Filtered && entry.ease_factor == 0;
```

Comment:
@dae, I add an extra condition to keep `early` reviews, because it affected the schedule.